### PR TITLE
[GM-835] Update documentation for environment variables

### PR DESCRIPTION
**Pull request by zionts**
Since the release of Apollo Server 2.13.0, the `APOLLO_KEY` and `APOLLO_GRAPH_VARIANT` environment variables are preferred to their `ENGINE_API_KEY` and `ENGINE_SCHEMA_TAG` predecessors. This updates the documentation for these, noting the former names for older versions of Apollo Server.

### CODE REVIEW

File: docs/source/tutorial/production.md [hunk 1]
```diff
@@ -31,24 +31,14 @@ The entry should basically look like this:
...
-  service: {
-    name: 'my-graph-name' // the name of your graph in Apollo Graph Manager
-  }
-};
-```
+Our key is now stored under the environment variable `APOLLO_KEY`.
 
 ### Check and publish with the Apollo CLI
 
 It's time to publish our schema to Graph Manager! First, start your server in one terminal window by running `npm start`. In another terminal window, run:
```
Comment (StephenBarlow) [inline docs/source/tutorial/production.md:38]: "In another terminal window, run the following command, providing the name of your graph where indicated:"

File: docs/source/tutorial/production.md [hunk 2]
```diff
@@ -31,24 +31,14 @@ The entry should basically look like this:
...
 
 ```bash
-npx apollo service:push --endpoint=http://localhost:4000
+npx apollo service:push --endpoint=http://localhost:4000 --graph=my-graph
```
Comment (StephenBarlow) [inline docs/source/tutorial/production.md:41]: Change "my-graph" to "name-of-graph"

File: docs/source/tutorial/production.md (hunk 1 above)
Comment (zionts) [inline docs/source/tutorial/production.md:38]:
```suggestion
It's time to publish our schema to Graph Manager! First, start your server in one terminal window by running `npm start`. In another terminal window, run the following command, substituting the name of your graph where indicated:
```

File: docs/source/tutorial/production.md (hunk 2 above)
Comment (zionts) [inline docs/source/tutorial/production.md:41]:
```suggestion
npx apollo service:push --endpoint=http://localhost:4000 --graph=name-of-graph
```

**Pull request merged**